From 03a0f368397b6258ad0fc5ad03676fb82f4a76a8 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Mon, 19 Apr 2021 10:10:25 +0200
Subject: [PATCH 0047/1695] scmi-msg: fix voltage domains inline comment header
 file

Fix voltage domains inline comment header file.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 015fe0ebf2388f84a129b7b04e6748e8fab0c01d scmi-msg: fix voltage domains inline comment header file]
Change-Id: I25ac7b23bfeacaa530a5efa3238434144e50d053
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne-lms: cherry-picked from a286b03f4e99344f3cade5fc01e6c1a6c3cc9e86]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/include/drivers/scmi-msg.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/include/drivers/scmi-msg.h b/core/include/drivers/scmi-msg.h
index 28961712f..c276ea853 100644
--- a/core/include/drivers/scmi-msg.h
+++ b/core/include/drivers/scmi-msg.h
@@ -334,7 +334,7 @@ int32_t plat_scmi_rd_set_state(unsigned int channel_id, unsigned int scmi_id,
 size_t plat_scmi_voltd_count(unsigned int channel_id);
 
 /*
- * Get clock controller string ID (aka name)
+ * Get voltage domain string ID (aka name)
  * @channel_id: SCMI channel ID
  * @scmi_id: SCMI voltage domain ID
  * Return pointer to name or NULL
@@ -381,7 +381,7 @@ int32_t plat_scmi_voltd_get_level(unsigned int channel_id, unsigned int scmi_id,
 /*
  * Set voltage domain level voltage domain
  * @channel_id: SCMI channel ID
- * @scmi_id: SCMI clock ID
+ * @scmi_id: SCMI voltage domain ID
  * @level: Target voltage domain level in microvolt
  * Return a compliant SCMI error code
  */
-- 
2.47.2

